src/Models/Income.cs

```diff
@@ -0,0 +1,12 @@
+namespace server.Models;
+
+public class Income
+{
+    public float Amount { get; set; }
+
+    public required string Date { get; set; }
```

Comment:
use `DateTime` as the type for dates with time (hour, minutes) and `DateOnly` for dates.